src/granite/base.cr

```diff
@@ -39,6 +39,29 @@ abstract class Granite::Base
   extend Integrators
   extend Select
 
+  @@last_write_time = Time.utc
```

Comment:
I thought about that. It can wait because its only costing us 1*number_of_models of queries when the app is launched. It seems like a such a tiny inefficiency. 

I'd say its low priority.